docs/installation/010_INSTALL_AUTOMATED.md

```diff
@@ -108,10 +108,11 @@ version of InvokeAI with the option to upgrade to experimental versions later.
     [latest release](https://github.com/invoke-ai/InvokeAI/releases/latest), and
     look for a series of files named:
 
-    - [InvokeAI-installer-2.2.4-p5-mac.zip](https://github.com/invoke-ai/InvokeAI/files/10254728/InvokeAI-installer-2.2.4-p5-mac.zip)
-    - [InvokeAI-installer-2.2.4-p5-windows.zip](https://github.com/invoke-ai/InvokeAI/files/10254729/InvokeAI-installer-2.2.4-p5-windows.zip)
-    - [InvokeAI-installer-2.2.4-p5-linux.zip](https://github.com/invoke-ai/InvokeAI/files/10254727/InvokeAI-installer-2.2.4-p5-linux.zip)
+    - InvokeAI-installer-2.X.X-mac.zip
+    - InvokeAI-installer-2.X.X-windows.zip
+    - InvokeAI-installer-2.X.X-linux.zip
```

Comment:
if we remove the version number from the installer package, it should be possible to link to the latest installer

Yes, I know, pros and cons.

But I would preffer the option to ask the user "did you download the latest installer" vs a user telling me "the zip I have on my system says vX.Y.Z 😅). Maybe include a .txt with the verison in the installer zip.

Reply:
I also agree that linking the latest installer would be a good change. But no link is better than a broken/outdated link, so this is a net positive. Hopefully in the near future the *installer* version won't matter at all.